inference/aligner.py

```diff
@@ -1303,6 +1349,42 @@ def chunkwise(patch_bbox):
       end = time()
       print (": {} sec".format(end - start))
 
+  def calc_res_and_compose(self, cm, model_path, src_cv, tgt_cv, vvote_field,
```

Comment:
Same thing as above. Would you update this to look like the other dataset methods? Call it `compute_field_and_vector_vote`, or something like that, too. In the refactor, I use `compose` only for adding vector fields together.